frontend/src/locales/en.yml

```diff
@@ -1375,3 +1376,52 @@ subscription:
       title: Enterprise Plan
       label: Early access
       desc: Large organization, has dedicated DBA group to manage database fleet
+engine:
+  mysql: MySQL
+  common: Common
+schema-review-policy:
+  title: Schema Review
+  no-policy-set: No schema review policy
+  add-review: Add policy
+  search-rule-name: Search rule name
+  remove-review: Successfully removed the review policy.
+  create-review: Successfully created the review policy.
+  update-review: Successfully updated the review policy.
+  filter: Filter
+  no-permission: Only DBA or Owner has permission to create or update review policy.
+  archive: Archive schema review policy
```

Comment:
From the UI, should be disable/enable instead of archive/restore

archive/restore is an implementation detail.